src/VueTyping.js

```diff
@@ -1,58 +1,102 @@
 export default {
-    render (h) {
-        return h('span', this.speechFrameFragment)
-    },
-    props: {
-        framerate: {
-            type: Number,
-            default: 1
+  render(h) {
+    return h("div", [
+      h("span", this.speechFrameFragment),
+      h(
+        "span",
+        {
+          style: {
+            opacity: this.cursorOpacity,
+            color: this.cursorAnimationType.color
+          }
         },
-        text: {
-            type: String,
-            default: ''
-        }
+        this.cursorAnimationType.cursor
+      )
+    ]);
+  },
+  props: {
+    framerate: {
+      type: Number,
+      default: 1
     },
-    data () {
-        return {
-            speechFrameFragment: ''
-        }
+    text: {
+      type: String,
+      default: ""
     },
-    watch: {
-        text (value) {
-            this.scheduleTyping()
-        },
-        framerate () {
-            this.scheduleTyping()
-        }
+    cursorAnimationType: {
```

Comment:
I think that "cursorAnimationType" is too long for a name. Maybe just "cursorOptions" is better.
If you can, please add this prop usage to the README and to the demo so users know it exists.

Reply:
There's another thing: how can the user disable the cursor?

Reply:
Good point, i will edit the README and add an option to disable the cursor, i totally forgot about it haha.